src/components/group/appsco-company-add-group.html

```diff
@@ -239,7 +248,9 @@
              * @private
              */
             _onFormError: function(event) {
-                this._showError(event.detail.request.response.message);
+                var code = event.detail.request.response.code;
```

Comment:
There is no need to create variable named 'code'. Pass it directly like this: 
****
this._showError(this.apiErrors.getError(event.detail.request.response.code));
****